js/index.js

```diff
@@ -0,0 +1,36 @@
+// Psuedo code
+// 1 - doc ready
+// 2 - make submit clickable via .submit
+// 3 - read the value entered into the field; store as var
+// 4 - make change all vals to lowercase
+// 5 - if-else to say if var is city name, then  render new background image
+
+
+$(document).ready(function() {
+	//console.log('got here');
+
+	$('#submit-btn').click(submit);
+
+
+	function submit (event) {
+		event.preventDefault();
+		var location = $('#city-type').val();
+		var location = location.toLowerCase().trim();
+		$('#city-type').val('');
+	
+
+
+	if (location === 'nyc' || location === 'new york' || location === 'new york city') {
```

Comment:
Great conditionals! I like that you made everything lowercase and then compared against those strings!